boards/nxp/frdm_mcxn236/board.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(CONFIG_SOC_MCXN236)
```

Comment:
Why make this dependent on the SOC Kconfig? I expect it will always be set for this board, so we can probably drop this right?

Reply:
Updated.